src/ErrorReporting/V1beta1/ErrorGroupServiceClient.php

```diff
@@ -89,7 +89,7 @@ class ErrorGroupServiceClient
     /**
      * The code generator version, to be included in the agent header.
      */
-    const CODEGEN_VERSION = '0.1.0';
+    const CODEGEN_VERSION = '0.0.5';
```

Comment:
It is being lowered because it is now being read from file in the generator directory, which is shared across languages: https://github.com/googleapis/toolkit/blob/master/version.txt The previous value was a hardcoded value specific to PHP.

Note also that this is no longer being included in the header - it is being replaced by the value we are reading from file.